cmd/download.go

```diff
@@ -125,6 +126,18 @@ func runDownload(cfg config.Config, flags *pflag.FlagSet, args []string) error {
 	}
 	fmt.Fprintf(Err, "\nDownloaded to\n")
 	fmt.Fprintf(Out, "%s\n", metadata.Dir)
+
+	// run postprocess script if one was configured
+	if usrCfg.GetString("postprocess") != "" {
+		cmd := exec.Command(usrCfg.GetString("postprocess"))
+		cmd.Dir = metadata.Dir
+		err := cmd.Run()
+		if err != nil {
+			fmt.Fprintf(Err, "\nThere was an error running the postprocess script: %v\n", err)
+		} else {
+			fmt.Fprintf(Err, "\nSuccessfully executed the postprocess script.\n")
+		}
```

Comment:
```suggestion
		if err == nil {
			fmt.Fprintf(Err, "\nSuccessfully executed the postprocess script.\n")
		} else {
			fmt.Fprintf(Err, "\nThere was an error running the postprocess script: %v\n", err)
		}
```

This focuses on the desired, and if that isn't the case, then it fails.  (Not tested).

Reply:
I tested this change locally and it was fine.